app/screens/post_options/post_options.tsx

```diff
@@ -63,17 +72,61 @@ const PostOptions = ({
     const shouldRenderFollow = !(sourceScreen !== Screens.CHANNEL || !thread);
     const shouldShowBindings = bindings.length > 0 && !isSystemPost;
 
-    const snapPoints = [
+    const snapPoints = useMemo(() => {
+        const items: BottomSheetProps['snapPoints'] = [1];
```

Comment:
These could all be pushed into the index file.

`managedConfig?.copyAndPasteProtection !== 'true';` 
* Daniel added an `observeConfigBooleanValue` function that could also be used for this one.

<img width="857" alt="image" src="https://user-images.githubusercontent.com/7575921/210712772-f3e184e3-d5c7-4a3d-b338-763e40e5fcd0.png">

Reply:
Nope this last one has nothing to do with the config that we get from the server, this is an EMM manage configuration and is retrieved by a hook